test/c_blog_server_test.js

```diff
@@ -0,0 +1,179 @@
+"use strict";
+
+const expect  = require('unexpected');
+const request = require('supertest');
+const baseUrl = 'http://localhost:3000';
+
+// Set mode to testing.
+process.env.TESTING = true;
```

Comment:
It looks like the package.json `test` script already sets `process.env.NODE_ENV = 'testing'` -- does this do something else?